index.html

```diff
@@ -0,0 +1,232 @@
+      <option value="AL">Alabama
+      <option value="CA">California
+      <option> … remaining US states … 
+    </optgroup>
+    <optgroup label="Canada">
+      <option value="AB">Alberta
+      <option value="CA">British Columbia
+      <option> … remaining provinces and territories … 
+    </optgroup>
+  </select>
+</div>
+      
+<div>
+  <label for="state" accesskey="s"></label>
+  <select name="state" id="state">
+    <option value="">United States
+    <optgroup label="USA">
+      <option value="AL">Alabama
+      <option value="CA">California
+      <option> … remaining US states … 
+    </optgroup>
+    <optgroup label="Canada">
+      <option value="AB">Alberta
+      <option value="CA">British Columbia
+      <option> … remaining provinces and territories … 
+    </optgroup>
+  </select>
+</div>
+
+<div class=" ">
+  <button type="button" class="btn btn-secondary">Upload Resume</button>
+</div>
+   
+<hr>
+<p class="p-centre"> By continuing you agree to Crowdstaffing's <a>Privacy policy</a> and <a>Terms of Use.</a>
+
+<div>
+  <button type="button" class="btn btn-info text-gray">SIGN UP TO CROWDSTAFFING</button>
+</div>
+      
+</div>
+</form>
+</div>
+
+<div class=" icon cl">
+  <div class="pl-8">
+  <img src="photo/buld.png" alt=""><p class="font">Access a who's who of employers across North America</p> 
+</div>
+
+<br>
+
+<div class="pl-8">
+  <img src="photo/sea.png" alt=""><p class="font"> Get matched to amazing roles based on your abilities and aspirations</p>
+</div>
+
+<br>
+
+<div class="pl-8">
+  <img src="photo/pot.png" alt=""><p class="font">Choose from full-time or contract options</p>
+</div>
+
+<br>
+
+<div class="pl-8">
+  <img src="photo/ch.png" alt="" id="lp-8"><p class="font">A talent advisor will walk you through every opportunity from  confirming fit to interview best practices</p>
+</div>
+
+<br>
+
+<div class="pl-8">
+  <img src="photo/globe.png" alt=""> <p class="font">The dynamic marketplace keeps contract workers <br> continuously re-assigned</p>
+</div>
+
+<br>
+</div>
+</section>
+
+
+<div class="pic cl">
+  <div class="pl">
+  <img src="photo/NBC-universal.png" alt="" id="img-down">
+  <img src="photo/SJI.png" alt="" id="img-down">
+  <img src="photo/KPMG.png" alt="" id="img-down">
+  </div>
+  <div class="pl">
+  <img src="photo/Radial.png" alt="" id="img-down">
+  <img src="photo/Illumina.png"alt="" id="img-down">
+  <img src="photo/Holman.png" alt="" id="img-down">
+  </div>
+</div>
+
+<section>
+  <div class="tag cl">
+    <a href="" id="tag"> <img src="photo/img-f.png" alt="" id="img-footer"></a>
+    <div class="a-tag">
+      <a href="" id="tag">For Businesses</a>
+      <a href="" id="tag"> For Staffing Agencies</a>
+      <a href="" id="tag">Job Seeker</a>
+      <a href="" id="tag">Blog</a>
+      <a href="" id="tag"> About Us</a>
+      <a href="" id="tag">Resources</a>
+      <a href="" id="tag">Privacy Policy</a>
+      <a href="" id="tag">  Terms & Conditions</a>
```

Comment:
@indalgit id always to be have unique name on a single page.. why you have used same id's for multiple elements ?